static/resources/harness.js

```diff
@@ -100,28 +100,25 @@
           }
         } else if (subtest.property == 'constructor') {
           var iface = parentPrefix+subtest.scope;
-          var args = '';
-          for (var a = 0; a < eval(iface+'.constructor.length'); a++) {
-            if (a == 0) {
-              args = args + '\'' + a + '\'';
-            } else {
-              args = args + ',' + '\'' + a + '\'';
-            }
-          }
 
           try {
-            result.code = 'new '+iface+'('+args+')';
-            eval('new '+iface+'('+args+')');
+            result.code = 'new '+iface+'()';
+            eval('new '+iface+'()');
             result.result = true;
           } catch (err) {
             if (
               err.name == 'TypeError' &&
               err.message == 'Illegal constructor'
             ) {
               result.result = false;
-            } else if (stringIncludes(err.message, 'Failed to construct')) {
-              // If it failed to construct, and not illegal, there's a
-              // constructor
+            } else if (
+              stringIncludes(err.message, 'Not enough arguments') ||
```

Comment:
Can you annotate which engine (and maybe version ranges?) uses these strings? Especially "Failed to construct" makes me curious.